.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,24 @@
+name: Lint Go Code
+
+on:
+  pull_request:
+    paths:
+      - '**/*.go'
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.19.0
+
+    - name: Run golangci-lint
+      uses: golangci/golangci-lint-action@v2
+      with:
+        version: latest
```

Comment:
By using the latest version, we can bring in breaking (or non-breaking) changes that will make the CI fail one day because of changes in the dependency.

I suggest pinning the version so the CI runs are stable and would not introduce any unexpected changes.